includes/models/class-accredible-learndash-model-auto-issuance.php

```diff
@@ -64,15 +66,15 @@ public static function validate( $data, $id = null ) {
 			$kind    = array_key_exists( 'kind', $data ) ? $data['kind'] : null;
 
 			if ( ! self::is_kind_valid( $kind ) ) {
-				wp_die( 'ERROR: ' . esc_attr( $kind ) . ' is an invalid kind.' );
+				throw new Exception( esc_attr( $kind ) . ' is an invalid kind.' );
```

Comment:
Could you also replace `wp_die` in the parent class `Accredible_Learndash_Model::validate`?

Reply:
Updated `Accredible_Learndash_Model::validate` and related tests.